components/crud-web-apps/jupyter/frontend/src/app/pages/index/index-default/index-default.component.ts

```diff
@@ -52,6 +54,7 @@ export class IndexDefaultComponent implements OnInit, OnDestroy {
     // Poll for new data and reset the poller if different data is found
     this.subs.add(
       this.poller.start().subscribe(() => {
+        //this.config = defaultConfig();
```

Comment:
What was this used for? Can it be removed?

Reply:
Removed.